javascript/javascript2/week1/hmw2.js

```diff
@@ -0,0 +1,25 @@
+const danishString = ["Jeg har en blå bil"];
+const danishString2 = ["Blå grød med røde bær"];
+function danish(danishString) {
+  const string = danishString.join();
+  let count = 0;
+  let obj = {};
+  for (let i = 0; i < string.length; i++) {
+    let strings = string[i];
```

Comment:
this is not actually a string...more like one character . check this https://www.w3schools.com/jsref/jsref_charat.asp
```suggestion
    let strings = string[i];
```